src/commands/force/source/manifest/create.ts

```diff
@@ -22,16 +22,22 @@ const manifestTypes: Record<string, string> = {
   package: 'package.xml',
 };
 
+const packageTypes: Record<string, string[]> = {
+  managed: ['installed', 'deprecated'],
+  all: ['installed', 'deprecated', 'installedEditable', 'deprecatedEditable'],
+};
+
 interface CreateCommandResult {
   name: string;
   path: string;
 }
 
-const xorFlags = ['metadata', 'sourcepath'];
+const xorFlags = ['metadata', 'sourcepath', 'targetusername'];
 export class create extends SourceCommand {
   public static readonly description = messages.getMessage('description');
   public static readonly examples = messages.getMessage('examples').split(os.EOL);
   public static readonly requiresProject = true;
+  public static readonly supportsUsername = true;
```

Comment:
This will cause issues with default usernames when this command runs, and we'd like to change the flag description to be specific for this command so we're suggesting to add another string type flag named, `fromorg` with no short char rather than use `supportsUsername`